src/V3Number.cpp

```diff
@@ -587,7 +587,7 @@ bool V3Number::displayedFmtLegal(char format, bool isScan) {
     }
 }
 
-string V3Number::displayPad(size_t fmtsize, char pad, bool left, const string& in) {
+string V3Number::displayPad(size_t fmtsize, char pad, bool left, const string& in) VL_MT_SAFE {
```

Comment:
PURE and likewise change .h file declaration.  I see others below like version(), I'm not going to mark them, please go through and look for others.